banzai/qc/pattern_noise.py

```diff
@@ -1,16 +1,17 @@
 import os
 import numpy as np
+from scipy.signal import cwt, ricker
+from scipy.ndimage.filters import median_filter
 
 from banzai.stages import Stage
 from banzai import logs
-from banzai.utils.stats import median_absolute_deviation
 
 
 class PatternNoiseDetector(Stage):
     # Signal to Noise threshold to raise an alert
-    SNR_THRESHOLD = 15.0
-    # Number of pixels that need to be above the S/N threshold to raise an alert
-    FREQ_BINS_THRESHOLD = 5
+    SNR_THRESHOLD = 10.0
+    # The maximum allowed standard deviation of the peak centres
+    STD_THRESHOLD = 1.0
```

Comment:
Is there a better name for this?

Reply:
I changed it to `PEAK_POSITION_STD_THRESHOLD`